src/System.Net.Mail/tests/Functional/SmtpExceptionTest.cs

```diff
@@ -86,24 +86,6 @@ public void TestConstructorWithStringArgument()
         }
 
         [Fact]
-        public void TestConstructorWithSerialization()
-        {
-            string msg = "MESSAGE";
-            Exception inner = new ArgumentException("whatever");
```

Comment:
Hmm, interesting.. I just found that you've already removed `[Serializable]` from `ArgumentException `. I'm not sure why test test still passes. But I think it would fail if we change its `GetObjectData` method.